### PR TITLE
Create DataTable Element | Table, Filter Tab, Filter Search, Pagination & fix responsive tables

### PR DESCRIPTION
**Pull request by kewcoder**
### Summary by CodeRabbit

- New Feature: Introduced a DataTable component that enhances data presentation with features like pagination, filtering, and bulk actions. This provides users with a more intuitive and efficient way to interact with large data sets.
- New Feature: Added a FormBuilder component that dynamically generates forms based on a JSON configuration, simplifying form creation and management.
- New Feature: Updated the Table component for better data display and interaction.
- New Feature: Improved the Pagination component for easier navigation through pages of data.
- New Feature: Enhanced the Input component for better user input experience.
- New Feature: Updated the App component with a new sidebar menu for improved navigation.
- New Feature: Added example DataTable components to demonstrate the new features and capabilities.
- New Feature: Updated the Header component for a better user interface.
- New Feature: Enhanced the RangeInput component for better user input experience.
- Chore: Updated the Tailwind CSS configuration for better styling and customization.

### CODE REVIEW

File: packages/@orchidui-vue/src/DataDisplay/Table/OcTable.vue [hunk 1]
```diff
@@ -1,121 +1,102 @@
...
-  isSelectable: Boolean,
+const emit = defineEmits({
+  "update:modelValue": [],
 });
-const selectedRows = ref([]);
-const queries = ref([]);
+
+const { isSelectable, fields, headers } = props.options;
+
+const selectedRows = ref(props.modelValue ? props.modelValue : []);
```
[the remark this answers is not in the document]
Comment (kewcoder) [reply packages/@orchidui-vue/src/DataDisplay/Table/OcTable.vue:16]: no need dynamic props

File: packages/@orchidui-vue/src/DataDisplay/Table/OcTable.vue [hunk 2]
```diff
@@ -1,121 +1,102 @@
...
-  fields: Array,
-  filterTabs: Array,
-  isSelectable: Boolean,
+const emit = defineEmits({
+  "update:modelValue": [],
 });
-const selectedRows = ref([]);
-const queries = ref([]);
+
+const { isSelectable, fields, headers } = props.options;
```
[the remark this answers is not in the document]
Comment (kewcoder) [reply packages/@orchidui-vue/src/DataDisplay/Table/OcTable.vue:14]: no need  reactive context props

File: package.json [hunk 3]
```diff
@@ -11,11 +11,10 @@
     "build-orchidui": "npm run build --prefix packages/@orchidui-vue",
     "build-docs": "npm run build --prefix docs",
     "build-playground": "vite build",
-    "build": "npm run build-orchidui && npm run build-docs && npm run build-playground && node obfuscator.js",
+    "build": "npm run build-playground",
```
Comment (mykhailo-diakovych) [inline package.json:14]: I think that the new build should also be applied to the vercel build, this may be the reason why the bill is falling?

Comment (kewcoder) [reply package.json:14]: maybe build cache https://vercel.com/docs/deployments/troubleshoot-a-build,

@spacemonkey92 can you share build error in vercel

**Review by mykhailo-diakovych — APPROVED** [review]

**Pull request merged**